CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ should change the heading of the (upcoming) version to include a major version b
 
 ## @rjsf/core
 - When rendering additional properties with title, use the key of the property instead of the title.
+  
+## @rjsf/playground
+- Replace Webpack with Vite
```

Comment:
Can you move this into the new `beta.9` branch under `## Dev / docs / playground` ?